src/solvers/dgsem_p4est/dg_2d_parabolic.jl

```diff
@@ -94,8 +94,20 @@ function rhs_parabolic!(du, u, t, mesh::P4estMesh{2},
                                        dg.surface_integral, dg)
     end
 
-    # TODO: parabolic; extend to mortars
-    @assert nmortars(dg, cache) == 0
+    # Prolong solution to mortars (specialized for AbstractEquationsParabolic)
+    # !!! NOTE: we reuse the hyperbolic cache here since it contains "mortars" and "u_threaded"
+    # !!! Is this OK?
```

Comment:
```suggestion
    # !!! NOTE: we reuse the hyperbolic cache here since it contains "mortars" and "u_threaded". See https://github.com/trixi-framework/Trixi.jl/issues/1674 for a discussion
```